config/schedule.rb

```diff
@@ -49,6 +49,11 @@
   rake "work:purge_old_drafts"
 end
 
+# Expire commenting for old admin posts
+every 1.day, at: "8:10 am" do
```

Comment:
We're [moving away from the whenever gem](https://otwarchive.atlassian.net/browse/AO3-5794) for scheduling. Can you add this to resque_schedule.yml instead?

#4280 is an example of how to do it. You can skip adding a rake as well.